src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -122,6 +123,19 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
+    /**
+     * Sort the list by alphabetical order.
+     *
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o1.toString().compareTo(o2.toString());
```

Comment:
Are you trying to sort by name or other attributes? I cannot get what you are comparing. If you just want to sort by name (as default), why not using `Collections.sort()` directly?